scipy/special/setup.py

```diff
@@ -92,21 +114,27 @@ def configuration(parent_package='',top_path=None):
                          **cfg)
 
     # Extension _ufuncs_cxx
-    ufuncs_cxx_src = ['_ufuncs_cxx.cxx', 'sf_error.c',
+    ufuncs_cxx_src = ['_ufuncs_cxx.cxx',
+                      'ellint_carlson_wrap.cxx',
                       '_faddeeva.cxx', 'Faddeeva.cc',
                       '_wright.cxx', 'wright.cc']
-    ufuncs_cxx_dep = (headers + ufuncs_cxx_src + cephes_src
-                      + ['*.hh'])
-    config.add_extension('_ufuncs_cxx',
-                         sources=ufuncs_cxx_src,
-                         depends=ufuncs_cxx_dep,
-                         include_dirs=[curdir] + inc_dirs,
-                         define_macros=define_macros,
-                         extra_info=get_info("npymath"))
+    ufuncs_cxx_dep = (headers + ufuncs_cxx_src + cephes_src +
+                      ['*.hh', join('ellint_carlson_cpp_lite', '*.hh')] +
+                      sf_error_src + sf_error_hdr)
+    ext_cxx = config.add_extension('_ufuncs_cxx',
+                                   sources=ufuncs_cxx_src,
+                                   depends=ufuncs_cxx_dep,
+                                   include_dirs=[curdir] + inc_dirs,
+                                   define_macros=define_macros,
+                                   libraries=["sf_error"],
+                                   extra_info=get_info("npymath"))
+    ext_cxx._pre_build_hook = cxx_pre_build_hook
 
     cfg = combine_dict(lapack_opt, include_dirs=inc_dirs)
+    cfg.setdefault('libraries', []).extend(["sf_error"])
     config.add_extension('_ellip_harm_2',
-                         sources=['_ellip_harm_2.c', 'sf_error.c',],
+                         sources=['_ellip_harm_2.c',],
```

Comment:
The lint test is failing, complaining about missing whitespace after the first comma on this line (the second comma is followed by a line break).

Reply:
Thank you for catching this rebase artefact. It is fixed in 5490f58b.